third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -26,4 +26,7 @@ impl Experiment {
     /// A flag which allows to turn off safety checks, like reference safety.
```

Comment:
This comment says reference safety is a permanent part of no-safety experiment, but this PR makes reference safety unconditional (not dependent on this flag).